01_fib/brnbke/main.go

```diff
@@ -0,0 +1,57 @@
+package main
+
+import (
+	"fmt"
+	"io"
+	"os"
+)
+
+var out io.Writer = os.Stdout
+
+// Returns Absolute vaule for a give integer
+func abs(x int) int {
+	if x < 0 {
+		return x * -1
+	}
+	return x
+}
+
+// Returns Fibonacci Number of given input n (-92 <= n <= 92) - where:
+// F0=0
+// for n > 0
+//   F1=1, Fn = Fn-1 + Fn-2
+// for n < 0H
+//   F-n=(-1)^(n+1)Fn.
+// max n = 92
+func fib(n int) {
+	if abs(n) > 92 {
+		fmt.Fprintln(out, `Fibonacci numbers greater than 92 not supported`)
```

Comment:
return here (low indentation level is easier to grok - you don't need the else branch).
you could also use a Panic.

Reply:
Good point, done